packages/mobile/src/screens/personal-info/styles.ts

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  main: {
+    flex: 1,
```

Comment:
It's widely used, let's take it to globalStyles as well, e.g.: globalStyles.flex1: { flex: 1 }.

Reply:
corrected